src/addonloader/luaaddonstate.h

```diff
@@ -123,6 +123,14 @@ class LuaAddonState {
     // @function currentInputMethod
     // @treturn string the unique string of current input method.
     DEFINE_LUA_FUNCTION(currentInputMethod);
+    /// Change the current input method
+    // @function setCurrentInputMethod
+    // @string str the input method name.
```

Comment:
this should be the "unique" name of input method, means that it's a string like "pinyin", not to be confused with the i18n'ed name